client-rpc/src/main.rs

```diff
@@ -30,12 +30,12 @@ pub(crate) struct Options {
     port: u16,
 
     #[structopt(
-        name = "chain-id",
+        name = "network-id",
         short,
         long,
-        help = "Chain ID (Last two hex digits of chain-id)"
+        help = "Network ID (Last two hex digits of network-id)"
```

Comment:
I guess the comment should be `"Network ID (Last two hex digits of chain_id)"`?
if network-id = the single byte convention;  chain-id = the full network name string in Tendermint's genesis.json

Reply:
ok